.travis.yml

```diff
@@ -20,14 +20,15 @@ matrix:
     env: RUN_PHPSTAN="TRUE"
 
 install:
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then wget https://github.com/phpstan/phpstan/releases/download/0.10.3/phpstan.phar; fi
+  - if [ $RUN_PHPSTAN == "TRUE"  ]; then composer require --dev phpstan/phpstan:^0.12; fi
 
 before_script:
   - composer update --prefer-source $LOWEST_DEPS
 
 script:
-  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/phpunit --coverage-clover=coverage.xml; fi
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then php phpstan.phar analyse -c phpstan.neon lib; fi
+  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/php-cs-fixer  fix --dry-run --diff; fi
```

Comment:
I ran php-cs-fixer on every version of PHP because it effectively does a "lint" syntax check "for free" - i.e. it will fail on syntax that is wrong for each/any of the PHP versions.

Reply:
cool!